scripts/cluster-helpers.sh

```diff
@@ -94,9 +94,12 @@ function native_terminate() {
 	echo
 	echo -n "###   Stopping HPO Service"
 	echo
-		
+
+	pkill -f service.py >/dev/null 2>&1
+	check_err "Failed to stop HPO Service!"
+
```

Comment:
Is pkill a standard package that is available on all distros? Why cant we use `kill` here ?

Reply:
Updated to 'kill' now